.golangci.yml

```diff
@@ -9,6 +9,9 @@ linters:
     - bodyclose
     - copyloopvar
     - staticcheck
+  exclusions:
+    paths:
+      - pkg/publicapi
```

Comment:
interesting that it's not complaining about the protobuff generated code. I guess that's fine.